### PR TITLE
Rahulbcn27 GitHub actions 1

### PR DESCRIPTION
**Pull request by rahulbcn27**
Migration from Travis CI to Github Actions Support.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Test-Build
```
Comment (rohityadavcloud) [inline .github/workflows/build.yml:1]: missing license @rahulbcn27

**Review by yadvr — CHANGES_REQUESTED** [review]
missing license

**Review by yadvr — APPROVED** [review]
LGTM - only way to test is to merge this. Merging this.

**Comment by yadvr** [comment]
Feedback @rahulbcn27 write better summary in your commit (i.e. commit message) and PR title/description.

**Pull request merged**